libkbfs/folder_branch_ops.go

```diff
@@ -493,13 +494,23 @@ func (fbo *folderBranchOps) deleteFromFavorites(ctx context.Context,
 	return favorites.Delete(ctx, h.ToFavorite())
 }
 
-// getHead should not be called outside of folder_branch_ops.go.
-func (fbo *folderBranchOps) getHead(lState *lockState) ImmutableRootMetadata {
+// getTrustedHead should not be called outside of folder_branch_ops.go.
```

Comment:
mention that it returns the empty `ImmutableRootMetadata` if the head is untrusted.